### DISCUSSION TITLE
Plugin Appreciation Thread <3

### DISCUSSION THREAD

**Discussion by mo-shaz** [williamboman/nvim-lsp-installer#564] [Show and tell]
You saved my day man ❤️.

I've only been developing frontend stuff with JavaScript/TypeScript and setting up tssserver was a hassle at the time. After that I forgot about it and everything was fine until I was to setup the lsp for Golang development and god I spent the whole afternoon to get the language server(the lsp client won't even detect the thing). I tried some other tools like "lsp-settings" and "vim-lsp" and those didn't seem to work either (I guess there are some issues with my Go installation path).

Saw your plugin mentioned in some reddit thread and my god what a relief. It didn't just solve my issue of finding a way to properly install the language-server, also sets up the thing so that I can be at peace knowing I didn't mess up the setup.

Thanks a lot man. These small things really do inspire junior devs like me. Have a great day 🥲.

**Comment by williamboman** [comment]
Hey, thanks for your kind words! I'm very happy to hear it's made neovim's builtin LSP client a bit more approachable for you, that's one of the main goals with this plugin!
